<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupWindow</class>
 <widget class="QDockWidget" name="StartupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaeRemote</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP-address, or name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server port</string>
    </property>
   </widget>
   <widget class="QListWidget" name="serverList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>641</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>310</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>StartupWindow</receiver>
   <slot>addServerToList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>StartupWindow</receiver>
   <slot>deleteServerFromList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>713</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>StartupWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>StartupWindow</receiver>
   <slot>connectToServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addServerToList()</slot>
  <slot>deleteServerFromList()</slot>
  <slot>connectToServer()</slot>
 </slots>
</ui>
